frigate/config/mqtt.py

```diff
@@ -15,6 +15,7 @@ class MqttConfig(FrigateBaseModel):
     enabled: bool = Field(default=True, title="Enable MQTT Communication.")
     host: str = Field(default="", title="MQTT Host")
     port: int = Field(default=1883, title="MQTT Port")
+    transport: str = Field(default="tcp", title="MQTT Transport Mechanism")
```

Comment:
I think it would be better if we added an enum class to MQTT and used that vs a manual field validator.